usvm-ts/src/test/kotlin/org/usvm/util/TSMethodTestRunner.kt

```diff
@@ -204,15 +207,15 @@ open class TSMethodTestRunner : TestRunner<TSTest, MethodDescriptor, EtsType?, T
             val fileURL = javaClass.getResource("/samples/${packagePath}/${id.fileName}")
                 ?: error("No such file found")
             val filePath = Paths.get(fileURL.toURI())
-            val file = loadEtsFileAutoConvert(filePath)
+            val scene = EtsScene(listOf(loadEtsFileAutoConvert(filePath)))
 
-            val method = file.getMethodByDescriptor(id)
+            val method = scene.files.single().getMethodByDescriptor(id)
```

Comment:
I'm not sure that it always contains only a single file

Reply:
Yeah, but this is just a jacodb API update fixup. Currently I don't want to write a correct search for a method among all scene files.